src/main/scala/nl/biopet/tools/vcfstats/VcfStats.scala

```diff
@@ -62,4 +62,49 @@ object VcfStats extends ToolCommand[Args] {
                maxContigsInSingleJob = Some(cmdArgs.maxContigsInSingleJob))
       .flatten
   }
+
+  def descriptionText: String =
+    """
+      | Vcfstats is a tool that can generate metrics from a vcf file.
+      | - General stats (default, can be disabled)
+      | - Genotype stats (default, can be disabled)
+      | - Sample compare (default, can be disabled)
+      | - Sample distributions (default, can be disabled)
+      | - Field histograms
+      |
+      | This tool can run locally single threaded but also on a Apache Spark cluster.
+    """.stripMargin
+
+  def manualText: String =
+    """
+      | Almost all stats are default but can be disabled if required. When having a lot of samples the sample compare step if very intensive.
+      |
+      | The reference fasta should have at least a dict file next to it.
+      |
+      | For the field histograms there are some methods to choose from. This method will be used to choose what value to report in the histogram if there multiple values. Some method require the value to be a Int or Double. If this is not the case an exception is thrown.
+      | - Min, takes minimal number (Int or double required)
+      | - Max, takes maximum number (Int or double required)
+      | - Avg, calculates the average (Int or double required)
+      | - All, returns all values as separated values
+      | - Unique, returns all unique values as separated values
+    """.stripMargin
+
+  def exampleText: String =
+    """
+      | Default vcfstats run:
+      | java -jar <vcfstats_jar> -I <input_vcf> -o <output_dir> -R <reference_fasta>
```

Comment:
Do not know how this shows up in markdown. The < > things.
Also to force code notation, prepend 4 spaces in front of java -jar. So
```
Bla
    java -jar
```
Renders as

bla
```
java-jar
```